android/app/src/main/python/mmrelay/setup_utils.py

```
@@ -0,0 +1,633 @@
+    Returns:
+        bool: True if the service is enabled, False otherwise.
+    """
+    try:
+        result = subprocess.run(
+            ["/usr/bin/systemctl", "--user", "is-enabled", "mmrelay.service"],
+            check=False,  # Don't raise an exception if the service is not enabled
+            capture_output=True,
+            text=True,
+        )
+        return result.returncode == 0 and result.stdout.strip() == "enabled"
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)